subsonic/api.go

```diff
@@ -242,7 +243,11 @@ func (connection *SubsonicConnection) GetMusicDirectory(id string) (*SubsonicRes
 func (connection *SubsonicConnection) GetRandomSongs() (*SubsonicResponse, error) {
 	query := defaultQuery(connection)
 	// Let's get 50 random songs, default is 10
-	query.Set("size", "50")
+	if viper.IsSet("client.random-songs"){
```

Comment:
Here we don't want to directly access viper's config because it couples us with code in stmps.go and this package (subsonic) should be decoupled as much as possible.
So to get the number of songs we use the variable we added to the struct at the top of this file instead.